python/docs/source/getting_started/quickstart_ps.ipynb

```diff
@@ -14486,4 +14486,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 1
-}
+}
```

Comment:
Is the newline removed automatically because I edited the file in IDE?

Reply:
Could be, I don't have strong feels about this whitespace change.